vendor-extra/JatsContentBundle/src/ViewConverter/FrontItemTagsVisitor.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\ViewConverter;
+
+use DOMNodeList;
```

Comment:
My IDE recommended that we add `ext-dom` to `composer.json`.

Reply:
And it is correct.

(I'd like to enable [ComposerRequireChecker](https://github.com/maglnet/ComposerRequireChecker), but have seen false positives when trying it before. Looks like there's config options now so might be doable.)